src/Access/GroupCheck.php

```diff
@@ -83,6 +83,10 @@ public function access(AccountInterface $user, Route $route, RouteMatchInterface
         return AccessResult::forbidden();
       }
 
+      if (is_string($group)) {
+        $group = entity_load($parameter_name, $group);
```

Comment:
FILE: /home/travis/build/Gizra/og/src/Access/GroupCheck.php
--------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
--------------------------------------------------------------------------------
 87 | WARNING | entity_load() calls should be avoided in classes, use
    |         | dependency injection and the "entity_type.manager" service
    |         | instead
--------------------------------------------------------------------------------